.github/workflows/updates/Minecraft Pi (Modded).sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# To avoid writing a custom updater script for this application, we will combine the separate version IDs used in the one URL as one large version variable.
+
+# Build ID
+# This obtains the latest successful build ID.
+build_id="$(curl https://jenkins.thebrokenrail.com/job/minecraft-pi-reborn/job/master/lastSuccessfulBuild/buildNumber)"
+
+# Version Number
+# This obtains the latest successful build's version number.
+deb_id="$(curl -s "https://jenkins.thebrokenrail.com/job/minecraft-pi-reborn/job/master/${build_id}/api/json?tree=artifacts%5BrelativePath%5D" | jq | grep 'client-.*-arm64.AppImage"' | sed 's/.*client-//g' | sed 's/-arm64.*//g')"
+
+# Update
+webVer="${build_id}/artifact/out/minecraft-pi-reborn-client-${deb_id}"
+# This URL is only used for validation.
+all_url="https://jenkins.thebrokenrail.com/job/minecraft-pi-reborn/job/master/${webVer}-armhf.AppImage"
```

Comment:
I just want to note, and this may have been entirely unintentional on your part, but the choice to get the deb_id using the `arm64` appimage and then use the `armhf` appimage for the all_url is a surprisingly good one.

This is because the all_url is verified by checking if that file actually exists online before creating the automated PR with it. Using arm64 for getting the version means that the arm64 version will be available, and then using the armhf for the all_url also checks that the armhf version is still available.